src/commands/help.js

```diff
@@ -0,0 +1,7 @@
+export default function(program) {
+  program
+    .command('help')
```

Comment:
I would add the alias for `h` too since you can do `--help` or `-h`. Also, can you match the description for this command to the `--help` flag? It's --> `output usage information`.